.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "cSpell.ignoreWords": ["menuselection"]
+}
```

Comment:
Please add a line with 
```
.vscode
```
to the `.gitignore` file (or possibly `.vscode/` - whichever one excludes the directory, to try to avoid further IDE-specific files being added later)